packages/react/src/modules/ms-teams/theme.css

```diff
@@ -1,4 +1,4 @@
-@import "@telegraph/tokens/light";
+@import "@telegraph/tokens/default";
```

Comment:
MS Teams uses light mode, but the guide devtools uses the dark styles, so we need to import the default tokens and use `data-tgph-appearance` to swap (MS Teams won't be affected bc the default is light).